client/src/components/questions/QAModule.jsx

```diff
@@ -2,20 +2,48 @@ import React, {useState, useEffect} from 'react';
 import StyleLib from '../../assets/Stylesheet.jsx';
 import api from '../../lib/api.js';
 import ReviewAndQuestionList from '../../assets/ReviewAndQuestionList.jsx';
+import QuestionList from './QuestionList.jsx';
+import SearchQA from './SearchQA.jsx';
 
-const QAModule = () => {
+const QAModule = ({product}) => {
+  const productID = product.id;
 
+  // React Hooks
+  const [numberOfQuestions, setNumberOfQuestions] = useState(2);
+  const [displayedResults, setDisplayedResults] = useState([]);
+  const [productResults, setProductResults] = useState([]);
+
+  // On Render Effects
+  useEffect(() => {getQuestions()}, []);
+  useEffect(() => {updateDisplayedResults()}, []);
+  useEffect(() => {updateDisplayedResults()}, [numberOfQuestions]);
+
+  // Helper Functions
+  const getQuestions = () => {
+    api.listQuestions({
+      product_id: productID,
+      page: 1,
+      count: 900
+    })
+      .then(data => {
+        setProductResults(data.results)
+        setDisplayedResults(data.results.slice(0, numberOfQuestions))
+      })
+      .catch(err => console.error(err))
+  }
+
+  const showMoreHandler = (e) => setNumberOfQuestions(numberOfQuestions + 2);
+  const updateDisplayedResults = () => setDisplayedResults(productResults.slice(0, numberOfQuestions));
+
+  // Rendered Elements
   return (
-    <div>
+    <StyleLib.tile>
       <StyleLib.h2>Questions and Answers</StyleLib.h2>
-      <StyleLib.h4>Yung Search Bar</StyleLib.h4>
-        <input type='text'/>
-      <StyleLib.h4>Example Question</StyleLib.h4>
-      <StyleLib.h5>Example Answer</StyleLib.h5>
-      <StyleLib.h5>Example Answer 2</StyleLib.h5>
-      <StyleLib.button>More Answered Questions</StyleLib.button>
+      <SearchQA />
+      <QuestionList displayedResults={displayedResults}/>
+      <StyleLib.button onClick={showMoreHandler}>More Answered Questions</StyleLib.button>
       <StyleLib.button>Add A Question</StyleLib.button>
-    </div>
+    </StyleLib.tile>
```

Comment:
Once we've got stuff working, let's try to refactor so we can share the Reviews and Questions list, and Reviews/Questions tiles. They are very very similar.